tests/certs/ca.crt

```diff
@@ -0,0 +1,20 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
We should not be committing certificates, even if I would much prefer it if that was possible: https://github.com/famedly/ldap-sync/issues/35#issuecomment-2321730986

Reply:
removed